src/Microsoft.Diagnostics.Runtime/src/DataTargets/ClrInfo.cs

```diff
@@ -33,21 +32,15 @@ internal ClrInfo(DataTarget dt, ClrFlavor flavor, ModuleInfo module, DacInfo dac
         /// </summary>
         public ClrFlavor Flavor { get; }
 
-        /// <summary>
-        /// Returns module information about the Dac needed create a ClrRuntime instance for this runtime.
-        /// </summary>
-        public DacInfo DacInfo { get; }
-
         /// <summary>
         /// Returns module information about the ClrInstance.
         /// </summary>
         public ModuleInfo ModuleInfo { get; }
 
         /// <summary>
-        /// Returns the location of the local dac on your machine which matches this version of Clr, or null
-        /// if one could not be found.
+        /// Target platform.
         /// </summary>
-        public string LocalMatchingDac { get; }
+        internal Platform Platform { get; }
```

Comment:
Should this be public?  Is it useful to know the target coredump/crashdump is from Linux/Windows?

Maybe not in the library's current state...

Reply:
marked internal in order to minimize changes to the public api at the current moment